packages/ui/src/primitives/card.tsx

```diff
@@ -54,6 +54,27 @@ export const Header = React.forwardRef<HTMLDivElement, React.HTMLAttributes<HTML
   );
 });
 
+export const Logo = React.forwardRef(function Logo(
+  {
+    className,
+    href,
+    ...props
+  }: React.ImgHTMLAttributes<HTMLImageElement> & {
+    href?: string;
+  },
+  forwardedRef: React.ForwardedRef<HTMLImageElement>,
+) {
+  const img = (
+    <img
+      crossOrigin='anonymous'
+      ref={forwardedRef}
+      {...props}
+      className={cn('max-h-24 max-w-24 object-contain', className)}
```

Comment:
Set arbitrary max-width and height for now. Previous implementation captured the width and height and applied a height to the wrapping container to prevent layout shift. We may need to figure something similar. It'd be nice if we somehow got the actual width and height back from the displayconfig in some fashion so we wouldn't have to do the calc client side.